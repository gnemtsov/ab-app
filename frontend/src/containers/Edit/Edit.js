import React from 'react';
import Form from '../../components/UI/Form/Form';

import classes from './Edit.css';

export default (props) => {
    const conf = {
        buttonText: ['Save', 'Saving..']
    }

    return (
        <div
            className={classes.FormContainer}>
            <Form
                conf={conf}
                title="Edit department"
                api={"/departments/edit" + props.location.search} />
        </div>
    );
};