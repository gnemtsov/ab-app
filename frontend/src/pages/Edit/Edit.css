.FormContainer {
    width: 40rem;
}