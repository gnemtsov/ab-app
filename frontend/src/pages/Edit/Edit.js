import React from 'react';
import Form from '../../containers/Form/Form';

import classes from './Edit.css';

export default (props) => {
    const conf = {
        buttonText: ['Save', 'Saving..']
    }

    return (
        <div
            className={classes.FormContainer}>
            <Form
                conf={conf}
                title="Edit department"
                api={"/departments/edit" + props.location.search} />
        </div>
    );
};