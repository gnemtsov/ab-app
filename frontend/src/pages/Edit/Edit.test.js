import React from 'react';

import { configure, render, mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Edit from './Edit';

configure({ adapter: new Adapter() });

describe('Edit', () => {
	it('Renders correctly', () => {
		const wrapper = shallow(<Edit location={{search: "?d_id=test"}} />);
		expect(wrapper).toMatchSnapshot();
	});
})
