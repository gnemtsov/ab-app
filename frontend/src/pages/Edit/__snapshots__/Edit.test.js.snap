// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Renders correctly 1`] = `
<div
  className="FormContainer"
>
  <Form
    api="/departments/edit?d_id=test"
    conf={
      Object {
        "buttonText": Array [
          "Save",
          "Saving..",
        ],
      }
    }
    title="Edit department"
  />
</div>
`;
